[16.01] Fix misplaced parenthesis

nsoranzo opened the pull request:
Reported by @dpryan79 in https://github.com/galaxyproject/tools-iuc/issues/906#issuecomment-238844304

nsoranzo committed:
Fix misplaced parenthesis

Reported by @dpryan79 in https://github.com/galaxyproject/tools-iuc/issues/906#issuecomment-238844304

mvdbeek commented on lib/tool_shed/galaxy_install/tool_dependencies/recipe/tag_handler.py:
```diff
@@ -418,9 +418,9 @@ def handle_complex_repository_dependency_for_package( self, elem, package_name,
-                        log.debug( "so appending it to the list of handled tool dependencies." )
+                        log.debug( "Tool dependency %s version %s cannot be installed (it was probably previously installed), "
+                                   "so appending it to the list of handled tool dependencies.",
+                                   str( tool_dependency.name), str( tool_dependency.version ) )
```
you're missing  `%`

dannon replied:
Not necessary using the `log.<level>` functions, string params are passed as extra arguments.

mvdbeek replied:
Good to know, thanks!

nsoranzo committed:
Pin flake8 version

Do not pin when forward-merging to dev branch.
Also ignore .tox/ and eggs/ as in #2650 .

nsoranzo commented:
I've added a commit to pin flake8 and ignore `.tox/` and `eggs/` directories, should fix py26-lint and py27-lint tests.

martenson commented:
thanks @nsoranzo !

nsoranzo commented:
Thanks @martenson, I'm forward merging now.
